requirements.txt

```diff
@@ -54,6 +54,7 @@ jmespath==0.9.3
 lazy-object-proxy==1.3.1
 MarkupSafe==1.0
 mccabe==0.6.1
+nose-timer==0.7.1
```

Comment:
add this to `requirements-top-level.txt`, then:

```
# Installing the top-level dependencies, upgrading them if necessary
pip install –r requirements-top-level.txt --upgrade 

# Updating requirements.txt
pip freeze –r requirements-top-level.txt > requirements.txt
```

Also please add the snip above somewhere we can reference in the future, thanks!